test/api/liskApi.js

```diff
@@ -399,6 +399,44 @@ describe('Lisk.api()', () => {
 			});
 		});
 
+		describe('#getDapp', () => {
+			it('should get a dapp by transactiondId', () => {
+				const transactionId = '7520138931049441691';
+				const options = {
+					transactionId,
+				};
+				LSK.getDapp(transactionId, callback);
+				(LSK.sendRequest.calledWithExactly(GET, 'dapps', options, callback)).should.be.true();
+			});
+		});
+
+		describe('#getDapps', () => {
+			it('should get dapps with options', () => {
+				const options = {
+					limit: defaultRequestLimit,
+					offset: defaultRequestOffset,
```

Comment:
They should get added soon: 
https://github.com/LiskHQ/lisk/issues/240#issuecomment-323074720